include/hwinfo/cpu.h

```diff
@@ -11,6 +11,24 @@ namespace hwinfo {
 
 class Socket;
 
+#ifdef HWINFO_UNIX
+struct Jiffies
+{
+  Jiffies(){
```

Comment:
This constructor is actually a default constructor. Use Jiffies() = default; instead